### PR TITLE
refactor(embed): deprecate addField

### PR DESCRIPTION
**Pull request by nyapat**
**Please describe the changes this PR makes and why it should be merged:**
should have been done before 14 released 

**Status and versioning classification:**
- I know how to update typings and have done so, or typings don't need updating
- Code changes have been tested against the Discord API, or there are no code changes

### CODE REVIEW

File: src/structures/MessageEmbed.js [hunk 1]
```diff
@@ -314,8 +315,17 @@ class MessageEmbed {
    * @param {string} value The value of this field
    * @param {boolean} [inline=false] If this field will be displayed inline
    * @returns {MessageEmbed}
+   * @deprecated This method is a wrapper for {@link MessageEmbed.addFields}, use that instead.
```
Comment (Jiralite) [inline src/structures/MessageEmbed.js:318]: The `.` references static methods.

```suggestion
   * @deprecated This method is a wrapper for {@link MessageEmbed#addFields}. Use that instead.
```

File: src/structures/MessageEmbed.js [hunk 2]
```diff
@@ -314,8 +315,17 @@ class MessageEmbed {
    * @param {string} value The value of this field
    * @param {boolean} [inline=false] If this field will be displayed inline
    * @returns {MessageEmbed}
+   * @deprecated This method is a wrapper for {@link MessageEmbed#addFields}. Use that instead.
    */
   addField(name, value, inline) {
+    if (!deprecationEmittedForAddField) {
+      process.emitWarning(
+        'MessageEmbed#addField is deprecated and removed in the next major. Use MessageEmbed#addFields instead.',
```
Comment (ArjunSharda) [inline src/structures/MessageEmbed.js:323]:
```suggestion
        'MessageEmbed#addField is deprecated and will be removed in the next major update. Use MessageEmbed#addFields instead.',
```

File: typings/index.d.ts [hunk 3]
```diff
@@ -1766,6 +1766,7 @@ export class MessageEmbed {
   public url: string | null;
   public readonly video: MessageEmbedVideo | null;
   public addField(name: string, value: string, inline?: boolean): this;
+  /** @deprecated This method is a wrapper for {@link MessageEmbed.addFields}, use that instead. */
```
Comment (ArjunSharda) [inline typings/index.d.ts:1769]:
```suggestion
  /** @deprecated This method is a wrapper for {@link MessageEmbed.addFields}. Use that instead. */
```

**Review by ArjunSharda — CHANGES_REQUESTED** [review]

File: typings/index.d.ts [hunk 4]
```diff
@@ -1766,6 +1766,7 @@ export class MessageEmbed {
   public url: string | null;
   public readonly video: MessageEmbedVideo | null;
   public addField(name: string, value: string, inline?: boolean): this;
+  /** @deprecated This method is a wrapper for {@link MessageEmbed#addFields}. Use that instead. */
```
Comment (almeidx) [inline typings/index.d.ts:1769]:
```suggestion
  /** @deprecated This method is a wrapper for {@link MessageEmbed#addFields}. Use that instead. */
  public addField(name: string, value: string, inline?: boolean): this;
```

**Review by almeidx — CHANGES_REQUESTED** [review]

**Review by ArjunSharda — APPROVED** [review]

**Review by kyranet — APPROVED** [review]

**Review by iCrawl — APPROVED** [review]

**Review by SpaceEEC — APPROVED** [review]

**Pull request merged**